arch/arm/core/aarch64/fatal.c

```diff
@@ -15,9 +15,20 @@
 
 #include <kernel.h>
 #include <logging/log.h>
+#include <exc_handle.h>
 
 LOG_MODULE_DECLARE(os, CONFIG_KERNEL_LOG_LEVEL);
 
+#ifdef CONFIG_USERSPACE
+Z_EXC_DECLARE(z_arm64_user_string_nlen);
+Z_EXC_DECLARE(arch_buffer_validate);
+
+static const struct z_exc_handle exceptions[] = {
```

Comment:
"Eventually" this could be done via linktime magic. This is fine for now though.